esclient.go

```diff
@@ -372,18 +372,13 @@ func buildQuery(launchName, uniqueID, logMessage string) interface{} {
 						"more_like_this": map[string]interface{}{
 							"fields":               []string{"message"},
 							"like":                 logMessage,
+							"min_doc_freq":         1,
 							"minimum_should_match": "90%",
 						},
 					},
 				},
 				"should": []map[string]interface{}{
 					{"term": map[string]interface{}{
-						"launch_name": map[string]interface{}{
```

Comment:
look. what if all tests in launch were failing due to some similar error. Like connection to DB is broken. 
Besides unique_id contains launch, we want to connect issue that we are investigating with other issues in the same launches. let's keep this boost